### PR TITLE
[WIP] posts: add post for the 0.10.1 release

### PR DESCRIPTION
**Pull request by fsouza**
Don't merge... yet!

### CODE REVIEW

File: _posts/2015-02-26-tsuru-server-0-10-1-is-out.markdown [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+categories:
+- Release
+tags:
+- release
+comments: []
+---
+
+
+Today we're releasing [tsuru-server 0.10.1](http://docs.tsuru.io/en/latest/releases/tsr/0.10.1.html), two days after the 0.10.0 release, to fix some potential bugs:
+
```
Comment (flaviamissi) [inline _posts/2015-02-26-tsuru-server-0-10-1-is-out.markdown]: That link is broken x)

Comment (fsouza) [inline _posts/2015-02-26-tsuru-server-0-10-1-is-out.markdown]: /me waiting for readthedocs to catch up.

Comment (flaviamissi) [inline _posts/2015-02-26-tsuru-server-0-10-1-is-out.markdown]: OH I see, okay then

Comment (fsouza) [inline _posts/2015-02-26-tsuru-server-0-10-1-is-out.markdown]: Ugh. I hate Read the docs. The tag didn't show up there it -__-'

Comment (fsouza) [inline _posts/2015-02-26-tsuru-server-0-10-1-is-out.markdown]: It's _now_ fixed. Took only 2 days, yay! -__-'

**Pull request merged**